visualisation/meshViewer.cpp

```diff
@@ -110,6 +110,45 @@ using namespace DGtal;
  */
 
 
+/**
+ * Custom Viewer3D to override KeyPressEvent method and handle new key display.
+ * It also desactivate the double Rendering mode for more efficiency.
+ **/
+template<typename Space = SpaceND<3>, typename KSpace = KhalimskySpaceND<3> >
+class CustomViewer3D: public Viewer3D<Space, KSpace>
```

Comment:
it just to be used here no need to add header ;) 
For the template the Space and KSpace are not defined as type in Viewer3D or Display3D so it not works ...